.rubocop.yml

```diff
@@ -78,3 +78,4 @@ Style/AccessorMethodName:
   Exclude:
     - 'lib/rx/**/*'
     - 'lib/sm/**/*'
+    - 'lib/mhv_logging/**/*'
```

Comment:
You can tell it to ignore those specific lines if you want.

`# rubocop:disable RULE`

We could also disable that rule, which IMHO makes no sense whatsoever.